tests/functional/cylc-reinstall/01-file-transfer.t

```diff
@@ -68,12 +69,13 @@ touch new_dir/new_file1 new_dir/new_file2
 rm -rf dir2-be-removed file2
 run_ok "${TEST_NAME}-reinstall" cylc reinstall "${RND_WORKFLOW_NAME}/run2"
 REINSTALL_LOG="$(find "${RND_WORKFLOW_RUNDIR}/run2/log/install" -type f -name '*reinstall.log')"
-grep_ok "
-del. dir2-be-removed/file1
-del. file2
-send new_dir/
-send new_dir/new_file1
-send new_dir/new_file2" "${REINSTALL_LOG}"
```

Comment:
This test was passing, whatever the contents of the reinstall log file, I expect this allowed the regression through.

Reply:
I'm not sure multiline usage is valid.

Reply:
Yes, `grep` will exit 0 if any line contained in the pattern matches. The first line in the pattern provided to `grep_ok` here is blank, so this was just testing that `REINSTALL_LOG` contained a newline character.

```console
$ grep '
> foo
> ''' <<< 'bar'; echo $?
bar
0
```